src/devices/yokis.ts

```diff
@@ -0,0 +1,2885 @@
+        manufacturerCode: Zcl.ManufacturerCode.YOKIS,
+        attributes: {
+            // This attribute represents the last value measured. The unit is in 0.01 °C (12,25°C -> 1225). Default: 0x0000, Min-Max: 0x954D - 0x7FFE. Reportable, not writable
+            currentValue: {ID: 0x0000, type: Zcl.DataType.INT16},
+            // Represent the minimal value set since the last power-on/reset. The unit is in 0.01 °C (12,25°C -> 1225). Default: 0x7FFE , Min-Max: 0x954D - 0x7FFE. Reportable, not writable
+            minMeasuredValue: {ID: 0x0001, type: Zcl.DataType.INT16},
+            // Represent the maximal value set since the last power-on/reset. The unit is in 0.01 °C (12,25°C -> 1225). Default: 0x954D , Min-Max: 0x954D - 0x7FFE. Reportable, not writable
+            maxMeasuredValue: {ID: 0x0002, type: Zcl.DataType.INT16},
+            // Represent the offset applicated to the temperature measured. The unit is in 0,1°C (1,5°C -> 15). Default: 0x00, Min-Max: 0xCE (-50) - 0x32 (50).
+            offset: {ID: 0x0003, type: Zcl.DataType.INT8},
+            // Represent the sampling period used to process the temperature measurement. The unit is in seconds. Default: 0x0A, Min-Max: 0x01 - 0x78.
+            samplingPeriod: {ID: 0x0004, type: Zcl.DataType.UINT8},
+            // Represents the sampling number to sense per sampling period defined before. Default: 0x03, Min-Max: 0x01 - 0x14.
+            samplingNumber: {ID: 0x0005, type: Zcl.DataType.UINT8},
+            // Represents the temperature variation to request a new temperature sending through reports. The unit is in 0,1°C (0,5°C ->5). Default: 0x00, Min-Max: 0x00 - 0x0A.
+            deltaTemp: {ID: 0x0006, type: Zcl.DataType.UINT8},
+            // Represents the minimal sending period that the device must respect before sending a new value through reporting. A writing on this attribute will update all reporting entries related to the temperature measurement. Default: 0x0A, Min-Max: 0x0000 - 0xFFFF.
+            minimalSendingPeriod: {ID: 0x0007, type: Zcl.DataType.UINT16},
+            // Represents the maximal sending period. The device must send a new value through reporting before the end of this period. A writing on this attribute will update all reporting entries related to the temperature measurement. Default: 0x0A, Min-Max: 0x0000 - 0xFFFF.
+            maximalSendingPeriod: {ID: 0x0008, type: Zcl.DataType.UINT16},
+        },
+        commands: {
+            // Reset the Min and Max temperature value information.
+            minMaxReset: {
+                ID: 0x00,
+                parameters: [],
+            },
+        },
+        commandsResponse: {},
+    },
+    manuSpecificYokisStats: {
+        ID: 0xfcf0, // Details coming soon
+        manufacturerCode: Zcl.ManufacturerCode.YOKIS,
+        attributes: {},
+        commands: {},
+        commandsResponse: {},
+    },
+};
+
+function deviceAddCustomClusters(clusterNames: string[]): ModernExtend {
```

Comment:
Please use the `modernExtend.ts` `deviceAddCustomCluster`

Reply:
Done : my function has very early when we introduced the support for Custom Clusters with the OnEvent.
I just have to use it several times per device because i declare between 5 to 6 custom clusters. But it should be fine still.